{
  "Faults": [
    {
      "py/object": "Fault",
      "loggedAt": "2022-05-16T17:32:00.000+0000",
      "latitude": 51.50896702394599,
      "estimatedRestoration": "2022-05-17T03:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JT2836'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JT2836",
      "reference": "JT2836",
      "engineerOnSiteTime": "2022-05-16T19:00:00.000+0000",
      "name": "UB3 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "UB3 3EP",
        "UB3 3ER",
        "UB3 3ES",
        "UB3 3HA",
        "UB3 3HB",
        "UB3 3HD",
        "UB3 3HE",
        "UB3 3HG",
        "UB3 3HL",
        "UB3 3JD",
        "UB3 3JE",
        "UB3 3JF"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -0.4140391529134606,
          51.50896702394599
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-05-16T18:30:00.000+0000",
      "UUID": "82ea79fc-fce6-4066-98fa-c2f8ba8de064",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.4140391529134606,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-05-16T21:40:00.000+0000",
      "latitude": 51.06526639226394,
      "estimatedRestoration": "2022-05-17T05:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JT2851'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JT2851",
      "reference": "JT2851",
      "engineerOnSiteTime": "2022-05-16T18:53:00.000+0000",
      "name": "SP1 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SP1 2DN",
        "SP1 2DR",
        "SP1 2SE",
        "SP1 2SF",
        "SP1 2SH",
        "SP1 2SX"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.7929087682530314,
          51.06526639226394
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-05-16T19:00:00.000+0000",
      "UUID": "6868a858-e3c9-4097-b5ab-23feaae1baab",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.7929087682530314,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-05-16T22:03:00.000+0000",
      "latitude": 51.0560845103575,
      "estimatedRestoration": "2022-05-17T01:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JT2899'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JT2899",
      "reference": "JT2899",
      "engineerOnSiteTime": "2022-05-16T23:20:00.000+0000",
      "name": "SP2 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SP2 8DB",
        "SP2 8DD",
        "SP2 8DE",
        "SP2 8NZ"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.8082021629705476,
          51.0560845103575
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-05-16T23:30:00.000+0000",
      "UUID": "7ab4e338-0eec-41df-8586-daf208432202",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.8082021629705476,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-05-16T22:31:00.000+0000",
      "latitude": 51.434529165491476,
      "estimatedRestoration": "2022-05-17T02:05:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JT2910'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JT2910",
      "reference": "JT2910",
      "name": "SN11 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SN11 0AB",
        "SN11 0HZ",
        "SN11 0JL",
        "SN11 0JX",
        "SN11 0SS",
        "SN11 8BA",
        "SN11 8DG",
        "SN11 8DH",
        "SN11 8DJ",
        "SN11 8DQ",
        "SN11 8EX",
        "SN11 8EY",
        "SN11 8HA",
        "SN11 8UH"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.9992177160779698,
          51.434529165491476
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-05-16T23:45:00.000+0000",
      "UUID": "8c1f96db-de27-43bb-b832-7a130e364993",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.9992177160779698,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-05-16T15:29:00.000+0000",
      "latitude": 51.0909927911292,
      "estimatedRestoration": "2022-05-17T05:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JT2771'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JT2771",
      "reference": "JT2771",
      "engineerOnSiteTime": "2022-05-16T17:00:00.000+0000",
      "name": "SP3 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SP3",
        "SP3 5AB",
        "SP3 5AD",
        "SP3 5AE",
        "SP3 5AF",
        "SP3 5AG",
        "SP3 5AH",
        "SP3 5AJ",
        "SP3 5AL",
        "SP3 5AN",
        "SP3 5AP",
        "SP3 5AQ",
        "SP3 5AR",
        "SP3 5AS",
        "SP3 5AT",
        "SP3 5AU",
        "SP3 5AW",
        "SP3 5AX",
        "SP3 5AY",
        "SP3 5AZ",
        "SP3 5BA",
        "SP3 5BB",
        "SP3 5BD",
        "SP3 5BE",
        "SP3 5BG",
        "SP3 5BH",
        "SP3 5BJ",
        "SP3 5BN",
        "SP3 5BQ",
        "SP3 5BS",
        "SP3 5BU",
        "SP3 5BW",
        "SP3 5BX",
        "SP3 5BY",
        "SP3 5DB",
        "SP3 5DU",
        "SP3 5FA",
        "SP3 5QP",
        "SP3 5QR",
        "SP3 5QS",
        "SP3 5QT",
        "SP3 5QU",
        "SP3 5QX",
        "SP3 5QY",
        "SP3 5R",
        "SP3 5RA",
        "SP3 5RB",
        "SP3 5RD",
        "SP3 5RE",
        "SP3 5RF",
        "SP3 5RG",
        "SP3 5RJ",
        "SP3 5RL",
        "SP3 5RQ",
        "SP3 5RS",
        "SP3 5RT",
        "SP3 5RU",
        "SP3 5TA",
        "SP3 5TE",
        "SP4",
        "SP4 6QX"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -2.0285643213862943,
          51.0909927911292
        ],
        "type": "Point"
      },
      "UUID": "4265052c-1ae4-41ff-af3b-d6f32ae5a20a",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "evChargersAffected": [
        "1c2bca26-ae6e-4bba-907a-200a2fbecc68",
        "c9148ae8-b231-4aa9-83c4-baac738e42ab",
        "15a69126-938c-4c67-a4df-76ee214d61de",
        "892d06c3-9d06-4df0-a862-1a1ee981b2ac",
        "89c6b6dc-b974-4bf6-8e72-425cf7728690"
      ],
      "longitude": -2.0285643213862943,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-05-16T22:09:00.000+0000",
      "latitude": 57.50702199724722,
      "estimatedRestoration": "2022-05-17T01:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on their way to site to get the power back on as quickly as they can. please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JT2902'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JT2902",
      "reference": "JT2902",
      "name": "AB42 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "AB42 1QE",
        "AB42 1QJ",
        "AB42 1SX",
        "AB42 1TA",
        "AB42 1TB"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.7849692695489978,
          57.50702199724722
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-05-16T22:59:00.000+0000",
      "UUID": "ea5fc571-34ce-44d4-af38-39107d49a842",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.7849692695489978,
      "resolved": false
    }
  ]
}
